llama_index/query_engine/sub_question_query_engine.py

```diff
@@ -70,7 +71,9 @@ def from_defaults(
         use_async: bool = True,
     ) -> "SubQuestionQueryEngine":
         callback_manager = None
-        if len(query_engine_tools) > 0:
+        if service_context is not None and service_context.callback_manager is not None:
```

Comment:
I don't think it's possible for the callback manager to be `None` on the service context

Reply:
You may be right! Will check on that and simplify this if I can